staging/src/k8s.io/kubectl/pkg/cmd/wait/wait.go

```diff
@@ -152,6 +157,7 @@ func (flags *WaitFlags) AddFlags(cmd *cobra.Command) {
 
 	cmd.Flags().DurationVar(&flags.Timeout, "timeout", flags.Timeout, "The length of time to wait before giving up.  Zero means check once and don't wait, negative means wait for a week.")
 	cmd.Flags().StringVar(&flags.ForCondition, "for", flags.ForCondition, "The condition to wait on: [delete|condition=condition-name[=condition-value]|jsonpath='{JSONPath expression}'=[JSONPath value]]. The default condition-value is true.  Condition values are compared after Unicode simple case folding, which is a more general form of case-insensitivity.")
+	cmd.Flags().BoolVar(&flags.WaitForCreation, "wait-for-creation", flags.WaitForCreation, "Default value is true as it waits for the creation of resources additionally, if it doesn't exist. This flag is ignored in --for=delete")
```

Comment:
I think this description sentence is a little off/confusing. Maybe format this description a little closer to the format of the other above:

"The default value is true. If set to true, also wait for creation of objects if they do not already exist. This flag is ignored in --for=delete"

Reply:
Thank you. Your suggestion is indeed more explanatory and I'll change that way.